pkg/customers/address.go

```diff
@@ -33,13 +33,14 @@ func createCustomerAddress(logger log.Logger, repo CustomerRepository) http.Hand
 			return
 		}
 
+		organization := route.GetOrganization(w, r)
```

Comment:
`route.GetOrganization` calls `moovhttp.Problem()` in its implementation
```go
	if ns := r.Header.Get(organizationHeaderKey); ns == "" {
		moovhttp.Problem(w, fmt.Errorf("missing %s header", organizationHeaderKey))
		return ""
    }
```
so we should check 
```go
if organization == "" {
  return
}
```

This isn't the best way to handle fetching the organization header but I'm hoping we'll be cleaning up this code in the near future.

Reply:
Good catch @vxio .  I'll add that